aea/aea.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         default_connection: Optional[PublicId] = None,
         default_routing: Optional[Dict[PublicId, PublicId]] = None,
         connection_ids: Optional[Collection[PublicId]] = None,
+        search_service_address: str = "",
```

Comment:
This should be configurable; not hard coded. Eventually, we need to simplify the AEA constructor.